File: FK.UI/web-ui-angular/src/app/models/Fridge.ts

export class Fridge {
  public id: number;
  public passwordhash: string;
  public name: string;
  public description: string;
  //public datecreate: date; 
}

export class FridgeView {
  public fridgeid: number; 
  public name: string;
  //public datecreate: date;
  public users: Array<any>
}

export class UserFridgeAdd {
  public id: number;
  public passwordhash: string; 
}

File: aspnetcore-angular/000my/000/CoreWebApi/CoreWebApi/FK.Models/DTO/Fridge.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.AspNetCore.Identity;

namespace FK.Models
{ 
    public partial class FridgeView
    {   
        public int Id { get; set; }
         
        public string PasswordHash { get; set; }
         
        public string Name { get; set; }
         
        public string Description { get; set; }

        public DateTime? DateCreate { get; set; } 

        public ICollection<StickerView> Stickers { get; set; }
        public ICollection<FridgeProductView> FridgeProducts { get; set; }
         
        public ICollection<UserFridgeView> UserFridges { get; set; } 
    }
}
